lib/pf/UnifiedApi.pm

```diff
@@ -242,6 +242,16 @@ our @API_V1_ROUTES = (
         Config::TrafficShapingPolicies
         Config::Violations
     ),
+    {
+        controller => 'Config::Switches',
```

Comment:
Shouldn't this namespace be removed from the list above since its now declared here?

Reply:
Yup!